docs/cli/getting-started/index.md

```diff
@@ -1,323 +1,505 @@
+- /stacks/first
+	[+] backend.tf
 
-```hcl
-# file: dev/dev.tm.hcl
+- /stacks/second
+	[+] backend.tf
 
-globals {
-  env = "dev"
-}
+Hint: '+', '~' and '-' mean the file was created, changed and deleted, respectively.
 ```
+:::
 
-```hcl
-# file: prod/prod.tm.hcl
+The generation report will report any changes in the generated code. When run twice, no changes will be made.
 
-globals {
-  env = "prod"
-}
+::: code-group
+```sh [shell]
+terramate generate
+```
+
+```sh [output]
+Nothing to do, generated code is up to date
 ```
+:::
 
-With this change, any stack under the prod directory will inherit a `global.env == "prod"` (unless overwritten or explicitly unset), and the same applies to `dev`. Now we want to import `mysite` code into the stack in each environment. In `<env>/mysite/stack.tm.hcl` insert the following:
+Let’s commit the changes and generated code:
 
-```hcl
-# file: <env>/mysite/stack.tm.hcl
+```sh
+git add stacks
+git commit -m 'Add a Terraform Backend Config to all stacks'
+```
 
-import {
-  source = "/modules/mysite/mysite.tm.hcl"
-}
+::: tip
+It's a recommended best practice to check in generated code to your repository.
+For details, please see [code generation best practices](../code-generation/index.md#best-practices).
+:::
+
+## Run Terraform in both stacks
+
+The stacks created in the previous sections both represent isolated Terraform environments. To make them functional,
+we must run `terraform init` in both. This is where the Terramate Orchestration comes in handy, which allows us to
+[run commands in stacks](../orchestration/run-commands-in-stacks.md) with the [run](../cmdline/run.md) command.
+As mentioned before, Terramate is not limited to orchestrating Terraform but can run any command.
+
+But before we can start, we need to prepare git to ignore terraform temporary files by adding a `.gitignore` file, which is
+located in the root directory of our repository:
+
+```sh
+# NOTE: 
+# You might not want to add state and lock file here
+# This is just convenient when running the quickstart guide
+cat <<EOF >.gitignore
+.terraform
+.terraform.lock.hcl
+terraform.tfstate
+EOF
+
+git add .gitignore
+git commit -m 'Add .gitignore'
 ```
 
-Update the output filename of the local_file resource in `/modules/mysite/stack.tm.hcl` to avoid overwriting between dev and prod. Use the Terramate metadata `${terramate.stack.path.relative}` in the path name:
+Now let’s initialize Terraform in all stacks:
 
-```hcl
-# in: modules/mysite/mysite.tm.hcl
-generate_hcl "mysite.tf" {
-  content {
-    resource "local_file" "mysite" {
-      filename = "/tmp/tfmysite/${terramate.stack.path.relative}/index.html"
-...
+::: code-group
+```sh [shell]
+terramate run terraform init
 ```
 
-To execute Terraform, navigate `"cd"` to `prod/mysite` and `dev/mysite` and run the necessary Terraform commands.
+```sh [output]
```

Comment:
The output looks a bit strange (the first word of each line is highlighted, see screenshot below). We could drop the `sh` from the code block to not use any colors in the output? 🤔 

![image](https://github.com/terramate-io/terramate/assets/92498/eaba71cc-df92-4f23-88d8-95dadb99d43d)

Reply:
changed this to text output instead - very good catch!